unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/UIText/Tests/UiTextHandlerShould.cs

```diff
@@ -65,6 +66,69 @@ public void CreateUiContainer()
                                .PutFor(scene, entity,
                                    Arg.Is<InternalUiContainer>(i => i.rootElement.Contains(handler.uiElement)
                                                                     && i.components.Contains(COMPONENT_ID)));
+
+            // check it initialized correctly
+            var style = handler.uiElement.style;
+            Assert.AreEqual(Position.Absolute, style.position.value);
+            Assert.AreEqual(LengthUnit.Percent, style.width.value.unit);
+            Assert.AreEqual(100, style.width.value.value);
+            Assert.AreEqual(LengthUnit.Percent, style.height.value.unit);
+            Assert.AreEqual(100, style.height.value.value);
+        }
+
+        [Test]
+        public void AdaptCorrectlyToUiTransformPadding ()
+        {
+            // 1. Set UiTransform padding style
+            var uiTransformHandler = new UITransformHandler(internalUiContainer, COMPONENT_ID+1);
+            PBUiTransform uiTransformModel = new PBUiTransform()
+            {
+                PaddingBottomUnit = YGUnit.YguPoint,
+                PaddingBottom = 25,
+                PaddingLeftUnit = YGUnit.YguPoint,
+                PaddingLeft = 15,
+                PaddingTopUnit = YGUnit.YguPoint,
+                PaddingTop = 35,
+                PaddingRightUnit = YGUnit.YguPoint,
+                PaddingRight = 50,
+            };
+            uiTransformHandler.OnComponentModelUpdated(scene, entity, uiTransformModel);
+
+            handler.OnComponentCreated(scene, entity);
+            handler.OnComponentModelUpdated(scene, entity, new PBUiText()
+            {
+                Value = "temptation",
+                TextAlign = TextAlignMode.TamMiddleCenter
+            });
+
+            // 2. Check that the UiText style adapted to its padding
+            var style = handler.uiElement.style;
+            Assert.AreEqual(uiTransformModel.PaddingBottom, style.bottom.value.value);
+            Assert.AreEqual(uiTransformModel.PaddingRight, style.right.value.value);
+            Assert.AreEqual(uiTransformModel.PaddingLeft, style.left.value.value);
+            Assert.AreEqual(uiTransformModel.PaddingTop, style.top.value.value);
```

Comment:
following _"one (logical) assertion per test"_ best practice, I would split this test in two. In the first one, I would have only these assertions without the 3. and 4. code in it, while second test will be similar to the current one but without these assertions.

Reply:
This fix and test have been removed since it was decided to be tackled with a different approach in another PR